src/modules/callouts/pages/CalloutsAdminPage.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <div class="mb-5 flex justify-between border-primary-40 border-b pb-3">
+    <PageTitle :title="t('menu.callouts')"></PageTitle>
+    <div class="flex-1 md:hidden">
+      <AppSelect v-model="currentStatus" :items="statusItems" />
+    </div>
+    <div class="flex-0 ml-3">
+      <AppButton href="/tools/polls">{{
```

Comment:
`href="/tools/polls"` 

is this right?

Reply:
It is for now until the create callout flow is ready